docs/root/configuration/other_protocols/thrift_filters/rate_limit_filter.rst

```diff
@@ -5,7 +5,7 @@ Rate limit
 
 * Global rate limiting :ref:`architecture overview <arch_overview_rate_limit>`
 * :ref:`v2 API reference <envoy_api_msg_config.filter.thrift.rate_limit.v2alpha1.RateLimit>`
-* This filter should be configured with the name *envoy.ratelimit*.
+* This filter should be configured with the name *envoy.filters.thrif.tratelimit*.
```

Comment:
Thanks! I think this one needs more adjusting, though. 😉

Reply:
oops.

Reply:
i'll change you your typo for mine :-)